airbyte-integrations/connectors/source-github/source_github/spec.json

```diff
@@ -36,11 +36,11 @@
             "title": "Authenticate with Personal Access Token",
             "required": ["personal_access_token"],
             "properties": {
-              "option_title": {
```

Comment:
this is a backwards incompatible change that will break the connector for all customers. We can't change property names  or the `const` values (but we can change `title` and `description` no problem)